src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --for-link: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo_img {
+  margin-top: 10px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav_ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_li {
+  margin-right: 20px;
+}
+
+.nav_li:last-child {
+  margin-right: 0;
+}
+
+.nav_a {
+  display: flex;
+  height: 60px;
+  font-weight: 500;
+  align-items: center;
+  text-decoration: none;
+  font-size: 12px;
+  color: #000;
+  text-transform: uppercase;
+}
+
+.is-active {
+  position: relative;
+  color: var(--for-link);
+}
+
+.nav_a:hover {
+  text-decoration: underline var(--for-link);
+}
+
+.blue:hover {
+  color: var(--for-link);
+  text-decoration: none !important;
+}
+
+.a::first-line {
```

Comment:
The selector `.a::first-line` seems incorrect. Ensure that `.a` is a valid class in your HTML, or adjust the selector to target the correct element.